src/platform/site-wide/cta-widget/CallToActionAlert.jsx

```diff
@@ -17,9 +19,18 @@ export default function CallToActionAlert({
     content: (
       <div>
         {alertText}
-        {buttonText && (
-          <button className={buttonClass} onClick={buttonHandler}>
-            {buttonText}
+        {primaryButtonText && (
+          <button className={buttonClass} onClick={primaryButtonHandler}>
+            {primaryButtonText}
+          </button>
+        )}
+        {secondaryButtonText && (
+          <button
+            className="va-button-link"
+            onClick={secondaryButtonHandler}
+            style={{ marginLeft: '1rem' }}
```

Comment:
Let's start using the padding utilities classes here, actually. You can add the class `vads-u-margin-left--1` for some standard spacing.